### DISCUSSION TITLE
Comments location

### DISCUSSION THREAD

**Discussion by mamodeo10** [requarks/wiki#5209] [Help / Questions] [answered]
Currently running wiki.js version: 2.5.170 on a docker container.
How do I export comments made on pages? When I create a backup and import it is only backing up pages. Where on the docker image would the location of the comments be?

**Comment by NGPixel** [comment, marked as answer]
This is not possible at the moment. Comments live in the DB so make sure to have regular backups of the DB and you'll be fine.

The ability to export comments to storage is planned for v3.

**Reply by mamodeo10** [reply]
Okay but where is the data stored on the DB…directory?

**Reply by NGPixel** [reply]
Which database are you using?

**Reply by mamodeo10** [reply]
MariaDB. Thanks for responding. Been waiting for someone. Thanks.

**Reply by mamodeo10** [reply]
How can I take comments linked to a page from a database and import them to a fresh docker container.
